quicklens/src/test/scala/com/softwaremill/quicklens/ModifyMapAtTest.scala

```diff
@@ -10,6 +10,12 @@ class ModifyMapAtTest extends AnyFlatSpec with Matchers {
     modify(m1)(_.at("K1").a5.name).using(duplicate) should be(m1dup)
   }
 
+  it should "modify a non-nested map with atOrElse" in {
+    modify(m1)(_.atOrElse("K1", A4(A5("d4"))).a5.name).using(duplicate) should be(m1dup)
+    modify(m1)(_.atOrElse("K1", ???).a5.name).using(duplicate) should be(m1dup)
```

Comment:
Testing that we don't call the default if we don't have to.

Reply:
👍 good idea